_integration-schemas/closeio/v1/activities.md

```diff
@@ -0,0 +1,347 @@
+    - name: _type
+      type: ['null', string]
+      description: <WRITE ME>
+    - name: attachments
+      type: array
+      decription: <WRITE ME>
```

Comment:
The value of `description` also needs to be a string: `"The customer's ID."`

If you need to use line breaks, you can do so like this, in which case quotes aren't necessary:

```
  - name: "id"
    type: "string"
    description: |
      The customer's ID.
```

This is also covered in the [metadocs SaaS table reference](https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas-table-schema/#table-attributes).